public/css/showoff.css

```diff
@@ -72,6 +72,15 @@
   }
 }
 
+.content {
+  position: relative;
+  /* for some reason, setting top to 50% is pushing the content too far down, no idea why */
+  top: 42%;
```

Comment:
No way for CSS to detect browser, it requires JS.  Can be done, if it is worth it.